mp4parse/src/lib.rs

```diff
@@ -829,8 +842,17 @@ struct ItemInfoEntry {
     item_type: u32,
 }
 
+/// See ISO 14496-12:2015 § 8.11.12
+#[derive(Debug)]
+struct ItemReferenceEntry {
+    item_type: FourCC,
+    from_item_id: u32,
+    to_item_id: u32,
```

Comment:
I've written it this way because a flattened structure is easier to create and easier to use. It's functionally equivalent to spec's structure, so there's nothing lost AFAIK.

I haven't seen files where one-to-many relationship would be actually needed, so I expect it to be pretty much always used with only a single `to` ID. Heap-allocating for just one 4-byte ID is wasteful, and creating `SmallTryVec` for just this case would be an overkill.

Reply:
Thanks for clarifying. I wonder if there might be a case like image sequences where a single alpha plane is applied to many different image planes. In any case, this seems fine and we can change it later if it proves necessary.